recipes/vs2008_runtime/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: vs2008_runtime
+  version: "9.0.30729.6161"
```

Comment:
@jakirkham, I updated to 9.0.30729.6161 which is the SP1 MFC security update

Reply:
Sounds fine to me. As long as we are equal to or greater than (which this is) `defaults` copy, I think we should be ok.